docs/designers-developers/designers/design-resources.md

```diff
@@ -1,6 +1,57 @@
+
+### How to get access
+To join the WordPress.org Figma team, ping in #design on chat.wordpress.org and let us know whether you need view-only or edit access. A Figma admin will contact you for your email address to set up your account.
+
+### Resources for learning how to use Figma
+[Getting started with Figma](https://help.figma.com/category/9-getting-started)
+
+[Top Online Tutorials to Learn Figma for UI/UX Design](https://medium.com/quick-design/top-online-tutorials-to-learn-figma-for-ui-ux-design-4e9c6721a72d)
+
+[Take a Tour Around Figma](https://help.figma.com/article/12-getting-familiar-with-figma)
+
+### Learning how to use files and projects
+[Getting started with Figma files and projects](https://help.figma.com/article/298-getting-started-with-files-and-projects)
+
+[What are files?](https://help.figma.com/article/298-getting-started-with-files-and-projects#files)
+
+[What are projects?](https://help.figma.com/article/298-getting-started-with-files-and-projects#projects)
+
+[Video tutorial](https://www.youtube.com/watch?v=c5HS6smhq2E)
+
+[FAQ](https://help.figma.com/article/298-getting-started-with-files-and-projects#faq)
+
+### Learning how to use components
+[Getting started with components](https://help.figma.com/article/66-components)
+
+[What are components?](https://help.figma.com/article/66-components#components)
+
+[Video tutorial](https://help.figma.com/article/66-components#videos)
+
+### Learning how to use WordPress Figma libraries
+**How to turn on the WordPress Components library in Figma**
+
+![How to turn on Figma libraries gif](https://wordpress.org/gutenberg/files/2019/08/figma-howtoturnonlibraries.gif)
+
+1. Click the **Team Library** icon in the **Assets** Panel: 
+
+![Hovering over the team library icon](https://wordpress.org/gutenberg/files/2019/08/figma-turn-on-libraries-e1564770916643.png)
+
+2. The **Library** modal will open and allow you to view a list of available libraries. Toggle to _Enable_ or _Disable_ a specific library:
+
+![Switching on the WordPress components libray in Figma](https://wordpress.org/gutenberg/files/2019/08/figma-libraries-e1564770879415.png)
+
+**How to refine or contribute to the WordPress components React library _(Coming soon)_**
+
+WordPress components in Figma mirror the live React components. Documentation for how to refine or contribute to WordPress components in React coming soon.
+
+## Sketch
+
+If you prefer to use [Sketch](https://www.sketch.com/), the [SketchPress](https://github.com/10up/SketchPress) project includes a library of Gutenberg design components helpful for designing and prototyping:
 
 [Download Sketch mockups & patterns files](https://github.com/10up/SketchPress)
```

Comment:
It doesn't look like these have been updated since February. Does it make sense to keep these included here?

Reply:
That's a good point. Might be best to remove this option in favor for the updated files.

Reply:
Removed.